src/dev/flang/be/c/Intrinsics.java

```diff
@@ -143,7 +143,20 @@ public class Intrinsics extends ANY
             );
         }
         );
-    put("fuzion.std.fileio.exists"       , noFileIo); // NYI: #158
+    put("fuzion.std.fileio.exists"       ,  (c,cl,outer,in) ->
+        {
+          var fileIdent = new CIdent("f");
+          return CStmnt.seq(
+            CExpr.decl("FILE *", fileIdent, CExpr.call("fopen", new List<>(A0.castTo("char *"),CExpr.string("r")))),
```

Comment:
you `fopen` a file without ever closing it. That is not so nice, you are leaking file handles and forbid opening the file for writing.

Reply:
@WaelYoussfi, I do not see where this is resolved. You need to close the files.